### DISCUSSION TITLE
Select a repository?

### DISCUSSION THREAD

**Discussion by bilogic** [microsoft/vscode-discussions#1176] [Extension Development QnA]
![image](https://github.com/microsoft/vscode-discussions/assets/946010/4ead8b09-3f2b-4366-b4ba-d2fe59d255b8)

I found a bug, but it isn't getting any attention https://github.com/microsoft/vscode/issues/206664

Is there an API to programmatically pick the git repository I want?

**Comment by isidorn** [comment]
I see the bug is fixed so I will go ahead and close this
